pinc/ProjectSearchResults.inc

```diff
@@ -650,9 +641,21 @@ class ProjectSearchResults
     }
 }
 
+function get_search_configure_link()
+{
+    $origin = urlencode($_SERVER['REQUEST_URI']);
+    return "<a href='{$_SERVER['PHP_SELF']}?show=config&amp;origin=$origin'>" . _("Configure Result") . "</a>";
+}
+
+function make_link($new_array, $text, $anchor = '')
+{
+    $url = "{$_SERVER['PHP_SELF']}?" . http_build_query(array_replace($_GET, $new_array)) . "$anchor";
```

Comment:
I realize this was in the code before your change, but we should avoid using `$_SERVER['PHP_SELF']` for security reasons. Odds are good you can just remove it completely and make this `"?" . http_build_query(...`